pkgs/by-name/li/librenms/package.nix

```diff
@@ -68,9 +68,8 @@ phpPackage.buildComposerProject rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  installPhase = ''
-    runHook preInstall
-
+  postInstall = ''
+    chmod -R u+w $out/share
```

Comment:
What is the reason for this `chmod`?

Reply:
With buildComposerProject2 you get permissions denied on mv or rm as at this stage code is readonly.